modules/end-user-guide/partials/proc_building-an-image-for-an-ide-based-on-the-intellij-platform.adoc

```diff
@@ -0,0 +1,160 @@
+[id="building-an-image-for-an-ide-based-on-the-intellij-platform_{context}"]
+= Building an image for an IDE based on the IntelliJ Platform
+
+This procedure describes how to build an image for an IDE based on the IntelliJ Platform up to version 2020.3. See the link:https://github.com/che-incubator/jetbrains-editor-images/blob/main/doc/Compatible-IDE.md[list of compatible IDEs].
+
+.Prerequisites
+
+* The build host has at least 2 GB of available RAM.
+
+* The build host has an installation of following tools:
+
+** link:https://www.docker.com/[Docker] version 18.09 or greater, supporting link:https://docs.docker.com/develop/develop-images/build_enhancements/[Docker BuildKit]
+
+** link:https://git-scm.com[Git]
+
+** link:https://www.gnu.org/software/libc/manual/html_node/Getopt.html[GNU getopt]
+
+** link:https://www.gnu.org/software/wget/[GNU wget]
+
+** link:https://openjdk.java.net/projects/jdk/11/[Java Development Kit (JDK) version 11]
+
+** link:https://stedolan.github.io/jq/[jq]
+
+
+.Procedure
+
+. Get a local copy of the link:https://github.com/che-incubator/jetbrains-editor-images[JetBrains Projector Editor Images repository].
++
+----
+$ git clone https://github.com/che-incubator/jetbrains-editor-images
+$ cd jetbrains-editor-images
+----
+
+. Run the build script with parameters:
++
+[subs="+quotes,macros,attributes"]
+----
+$ ./projector build --tag __<tag>__ --url __<URL>__
+----
++
+`--tag __<tag>__`::
+The name and tag to apply to the image after build in `name:tag` format.
++
+`--url __<URL>__`::
+The URL pointing to an archive of the custom IntelliJ-based distribution. The editor version can be at most 2020.3 because Projector doesn't support later versions. The archive must target the Linux platform, be in `+tar.gz+` format, and include JetBrains Runtime (JBR). 
```

Comment:
```suggestion
The URL pointing to an archive of the custom IntelliJ Platform-based IDE distribution. The editor version can be at most 2020.3 because Projector doesn't support later versions. The archive must be a `+tar.gz+`, target the Linux platform, and include JetBrains Runtime (JBR). 
```

Reply:
A Platform-based contraction that I forgot to replace.

The order matters: 1.  platform, 2. format, 3. content. It's the natural order when searching for an archive. When searching in JetBrains of Android Studio download website, you will find the information in that order. 

 "a  tar.gz" looks awkward.  `tar.gz` is really a file format.

Reply:
> "a tar.gz" looks awkward. tar.gz is really a file format.

+1